harmonizer/types.ts

```diff
@@ -103,6 +103,8 @@ export type Artwork = {
 	types?: ArtworkType[];
 	comment?: string;
 	provider?: ProviderName;
+	/** Whether to set `referrerpolicy="no-referrer"` for the artwork URLs */
+	noReferrer?: boolean;
```

Comment:
I think I would prefer to set `<meta name='referrer' content='same-origin' />` in `server/routes/_app.tsx` rather than setting this per image.